src/main/scala/io/citrine/lolo/trees/classification/ClassificationTrainingLeaf.scala

```diff
@@ -1,19 +1,19 @@
 package io.citrine.lolo.trees.classification
 
-import io.citrine.lolo.{Learner, TrainingResult}
+import io.citrine.lolo.{Learner, TrainingResult, TrainingRow}
 import io.citrine.lolo.trees.TrainingLeaf
 import io.citrine.random.Random
 
 import scala.collection.mutable
 
 /** Training leaf for a classification tree. */
 case class ClassificationTrainingLeaf(
-    trainingData: Seq[(Vector[AnyVal], Char, Double)],
```

Comment:
One thing to note here. Because `TrainingRow` contains `inputs: Vector[Any]`, we are sacrificing the typing within trees that the inputs are a `Vector[AnyVal]` after encoding. 

Seeing as the distinction between `Any` and `AnyVal` doesn't really offer us much here, I think this is a fair trade to use the same `TrainingRow` class in all locations.

Reply:
Why can't we enforce `inputs: Vector[AnyVal]` throughout? Do we allow an input entry to be a sequence?

Reply:
A `String` is not an `AnyVal`, is the main reason.